pkg/features/kube_features.go

```diff
@@ -1992,6 +1992,10 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 		{Version: version.MustParse("1.34"), Default: true, PreRelease: featuregate.Beta},
 	},
 
+	kcmfeatures.CloudControllerManagerWatchBasedRoutesReconciliation: {
+		{Version: version.MustParse("1.35"), Default: false, PreRelease: featuregate.Alpha},
+	},
+
```

Comment:
is this feature gate really needed?

Reply:
IIRC if this is only cloud provider specific you can just use the feature gate from the cloud provider, but you should check